### PR TITLE
Oz

### PR DESCRIPTION
**Pull request by ozgemer**
@Sa-ar

### CODE REVIEW

File: First-Html-page/personal-site-challenge.html [hunk 1]
```diff
@@ -7,23 +7,63 @@
...
-        LinkedIn
-        Twitter
-        Website
-        © DevClub
+        
+        <div class='header'>
+            <div class='container'>
+                <h1 class='header-text'>I'm a Dev Club Member.</h1>
+                <h2 class='header-text'>a <u>pro</u>grammer</h2>
+                <img id='mountain-img' src='asset/mountain.png'/>
```
Comment (Sa-ar) [inline First-Html-page/personal-site-challenge.html:19]: why not use classes?

File: First-Html-page/personal-site-challenge.html [hunk 2]
```diff
@@ -7,23 +7,63 @@
...
+            <img src='asset/cloud.png' id='cloud-1'/>
+            <img src='asset/cloud.png' id='cloud-2'/>
+                                              
+        </div>
+        <div class='main_text'>
+            <div class='container'>
+                <img src='asset/profile-pic.png' id='profile-img'/>
+                <h2>Hello.</h2>
+                <p>I am a Web Developer.And I ❤️ to code</p>
+                <hr>
```
Comment (Sa-ar) [inline First-Html-page/personal-site-challenge.html:30]: very good!

File: First-Html-page/personal-site-challenge.html [hunk 3]
```diff
@@ -7,23 +7,63 @@
...
+                <div class='clear'></div>
+                <div class='subjects-div'>
+                    <img src='asset/comp2-gif.gif' id='gif-2'/>
+                    <div id='debug-div'>
+                        <h3>Debugging</h3>                    
+                        <p id='debug-p'>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quis illum autem harum molestiae necessitatibus impedit placeat, rem consectetur provident. Ex laudantium odit fugit quo aliquam corporis nulla quaerat quas incidunt?
+                        </p>
+                    </div>
+                </div>
+                <div class='clear'></div>
```
Comment (Sa-ar) [inline First-Html-page/personal-site-challenge.html:49]: what is this for?

File: First-Html-page/style.css [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+body{
+    margin:0;
+    font-family: 'Merriweather';
+    font-size:small;
+}
+h1{
+    font-family: 'Sacramento';
+}
+h2,h3,h4,h5,h6,h7{
```
Comment (Sa-ar) [inline First-Html-page/style.css:13]: Nice!

File: Fylo-FronedMentor-Challenge/index.html [hunk 5]
```diff
@@ -7,16 +7,37 @@
...
+        <img class='upload-imgs' src='./assets/icon-document.svg'/>
+      </span>
+      <span class='upload-logos'>
+        <img class='upload-imgs' src='./assets/icon-folder.svg'/>
+      </span>
+      <span class='upload-logos'>
+        <img class='upload-imgs' src='./assets/icon-upload.svg'/>
+      </span>
+    </div>
+    <div class='flexbox-item flexbox-item-2'>
```
Comment (Sa-ar) [inline Fylo-FronedMentor-Challenge/index.html:29]: try naming you classes better

**Review by Sa-ar — CHANGES_REQUESTED** [review]
Try to use more classes and fewer ids for styling purposes

File: First-Html-page/personal-site-challenge.html (hunk 3 above)
Comment (ozgemer-git) [reply First-Html-page/personal-site-challenge.html:49]: I used floats for the GIF images, the clear div makes the display appear as wanted

File: First-Html-page/personal-site-challenge.html (hunk 1 above)
Comment (ozgemer-git) [reply First-Html-page/personal-site-challenge.html:19]: I thought you should use classes when you want to reuse css and ids when its for a specific element.
If its better to use classes generally, when should i use ids?

Comment (Sa-ar) [reply First-Html-page/personal-site-challenge.html:19]: That's a great question. try to search for the answer and send me what you discovered on slack

File: First-Html-page/personal-site-challenge.html (hunk 3 above)
Comment (Sa-ar) [reply First-Html-page/personal-site-challenge.html:49]: That's OK. Try to use other technics that are more modern next time

**Pull request closed without merging**